libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationActivity.java

```diff
@@ -69,8 +71,10 @@ protected void onDestroy() {
   public void onNavigationReady() {
     NavigationViewOptions.Builder options = NavigationViewOptions.builder();
     options.navigationListener(this);
-    extractRoute(options);
-    extractCoordinates(options);
+    if (!isConfigurationChange) {
```

Comment:
@ericrwolfe this prevents the UI from calling `MapboxNavigation#startNavigation` twice (we already have a cached route in the `RouteViewModel`)